src/System.Private.CoreLib/shared/System/Runtime/CompilerServices/FixedAddressValueTypeAttribute.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.CompilerServices
+{
+    [Serializable]
+    [AttributeUsage(AttributeTargets.Field)]
+    public sealed class FixedAddressValueTypeAttribute : Attribute
+    {
+        public FixedAddressValueTypeAttribute() { }
```

Comment:
> This attribute affects GS cookie generation

The FixedAddressValueTypeAttribute affects whether the statics are pinned. I do not think it affects GS cookie generation. For CoreRT, throwing PNSE for it should be acceptable.

The attribute that affects GC cookie generation is UnsafeValueTypeAttribute.

We should have tracking issues for both.